src/deriver/deriver.py

```
@@ -120,144 +47,164 @@
 
-    async with tracked_db("deriver.get_session_context_formatted") as db:
-        formatted_history = await summarizer.get_session_context_formatted(
-            db,
-            latest_message.workspace_name,
-            latest_message.session_name,
-            token_limit=available_context_tokens,
-            cutoff=earliest_message.id,
-            include_summary=True,
+        logger.debug(
+            "Token estimation - Peer card: %d, Working rep: %d, Sys prompt: %d, Usr prompt: %d, "
+            + "New turns: %d, Total estimated: %d, Available for context: %d",
```

Ruff (0.14.2)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)